rosdep/python.yaml

```diff
@@ -4136,6 +4136,10 @@ python-sqlalchemy:
     wily: [python-sqlalchemy]
     xenial: [python-sqlalchemy]
     xenial_python3: [python3-sqlalchemy]
+python-sqlite:
+  debian: [python-sqlite]
+  fedora: [python-sqlite3]
```

Comment:
This isn't a Fedora package. Your packaging link is for EPEL. Fedora 31 will drop Python 2 anyway, so we should just omit the Fedora rule.

Reply:
Thanks, updated.